packages/preview-middleware/test/unit/base/__snapshots__/flp.test.ts.snap

```diff
@@ -489,7 +489,7 @@ exports[`FlpSandbox router editor with config 1`] = `
     <!-- Bootstrap the UI5 core library. 'data-sap-ui-frameOptions=\\"allow\\"'' is a NON-SECURE setting for test environments -->
     <script id=\\"sap-ui-bootstrap\\"
         src=\\"../resources/sap-ui-core.js\\"
-        data-sap-ui-libs=\\"sap.m,sap.ui.core,sap.ushell,sap.f,sap.ui.comp,sap.ui.generic.app,sap.suite.ui.generic.template,sap.ui.rta\\"
+        data-sap-ui-libs=\\"sap.m,sap.ui.core,sap.ushell,sap.f,sap.ui.comp,sap.ui.generic.app,sap.suite.ui.generic.template,sap.ui.rta,sap.fe.core,sap.fe.templates\\"
```

Comment:
oh my ... that list is bad. I think there is was an issue even before your change. The v2 libs should only be there in case of v2 and the v4 libs in case of v4.

Reply:
This looks good again after reverting the change in the `flp.ts`. This test result is correct, however, it might make sense to modify the test to be more realistic.

Reply:
Fixed with 82aa6ccd58d35ede39c5fb2e786f978eee3665a7